lib/config.js

```diff
@@ -376,7 +415,7 @@ function create_iobroker_object(id, role, type, dataType, states, writable, text
 }
 
 function determine_unit(key) {
-    if (key.indexOf("_TEMP_") > 0)
+    if (key.indexOf("_TEMP_") > 0 && key.indexOf("_MODE") < 0)
```

Comment:
i think we should eliminate that function completely and provide the unit for each object. I understand your workaround, if we do not refactor that in this pull request we should do it in another in near future.

Reply:
I also think an unit per object would be the best solution.

Unfortunatly the workaround doesn´t fix the problem, maybe you´ve got another idea for a quick solution? VlxDevConstants.C_CYC_OPT_NONE is still always changed to -273

If not, we should leave A_CYC_OPT_TEMP_SENSOR_MODE commented out (as it is the pull request) and add it when we support units.